lychee-lib/src/types/input.rs

```diff
@@ -260,6 +260,40 @@ impl Input {
         }
     }
 
+    /// Retrieve all sources from this input. The output depends on the type of
+    /// input:
+    ///
+    /// - Remote URLs are returned as is, in their full form
+    /// - Filepath Glob Patterns are expanded and each matched entry is returned
+    /// - Absolute or relative filepaths are returned as is
+    /// - All other input types are not returned
```

Comment:
Maybe we could print `Stdin` for standard input and `Raw String` for strings? This could avoid some confusion when people try to troubleshoot their inputs. I don't have a strong opinion on this, though, so feel free to ignore.

Reply:
Yes, I was also thinking about this. I don't have a super strong opinion on this as well, but I can print out `Stdin` and `Raw String` as you suggested.

Reply:
Added in 07fe746